docs/source/overview/installation.rst

```diff
@@ -84,27 +72,19 @@ needs to be one of the following:
 
 .. _install from package:
 
-Install from prepared package
+Install from prepared packages
 -----------------------------
 
-To install SlaxWeb Framework from a prepared package, you will first have to download
-it, from the `GitHub Release pages`_, or find the appropriate download link, a little
-bit down this document. When installing from a prepared package, there currently
-is no way of installing the latest development snapshot, and you will have to use
-the composer from :ref:`install with composer` section above.
-
-To begin installation we are first going to download the desired version package
```

Comment:
Hi @slax0rr , I have made the required changes and pushed a new commit. Please verify.